### ISSUE TITLE
Add the `./src/` folder to exports

### ISSUE THREAD

**Issue by kwangure** [argyleink/open-props#186]
At present, the only way to use Open Props in a `"type": "module"` setup it is through the `"module": "open-props.module.js"` file which slightly impairs the ability of bundlers to optimize the package since it's operating on a pre-combiled bundled. It would be helpful if setups with bundlers could directly import the original `./src/index.js`. 

Furthermore, I'd like to also to import the individual files into js e.g `import { Teal } from "./src/props.colors.js"` and have my bundler take care of the treeshaking etc. rather than import the whole `open-props.module.js`. 

This is a feature request to export the `src`folder:

```diff
"exports": {
    ".": {
      "import": "./dist/open-props.module.js",
      "require": "./dist/open-props.cjs",
      "default": "./dist/open-props.cjs"
    },
+   "./src": "./src",
	"./style": "./open-props.min.css",
```

### Additional Context
Some bundlers (e.g Vite) strictly only use the `exports` field if it exists. That is, acessing the `./src` folder throws '`./src` is not exports'.

**Comment by argyleink** [comment]
cool, agree that it'd be nice to get access to the individual files from JS as well as offer the src module. 

is that exports addition to package.json all that's required!? bundlers will resolve files in src even if not explicitly in exports?

do you want to PR the change? i think this can be done soon tho 👍🏻

**Comment by argyleink** [comment]
fixed in `v1.3.6` 🎉
